src/main/kotlin/ecommerce/repository/CartRepository.kt

```diff
@@ -0,0 +1,201 @@
+package ecommerce.repository
+
+import ecommerce.dto.CartItemDto
+import ecommerce.exception.NotFoundException
+import org.springframework.jdbc.core.simple.JdbcClient
+import org.springframework.stereotype.Repository
+
+@Repository
+class CartRepository(private val jdbcClient: JdbcClient) {
+    fun addItemToCart(
+        productId: Long,
+        productQuantity: Long,
+        cartId: Long,
+    ): CartItemDto {
+        val updateSql = """
+        UPDATE cart_items
+        SET quantity = quantity + ?
+        WHERE product_id = ? AND cart_id = ?
```

Comment:
It names `CartRepository`, but it handles `CartItem`.
What's different of two words?